v4-client-js/package.json

```diff
@@ -36,7 +36,7 @@
     "@cosmjs/stargate": "^0.30.1",
     "@cosmjs/tendermint-rpc": "^0.30.1",
     "@cosmjs/utils": "^0.31.1",
-    "@dydxprotocol/v4-proto": "^0.2.1",
+    "@dydxprotocol/v4-proto": "^0.3.0-dev.1",
```

Comment:
Only concern here is if we bump v4-proto again, the bridge feature would be lost.

Reply:
umm yea will resync and see we can actually publish a non dev version, otherwise doing it to unblock dev work for now